configreader/dumper_test.go

```diff
@@ -0,0 +1,52 @@
+package configreader
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"testing"
+	"time"
+
+	v3 "github.com/envoyproxy/go-control-plane/envoy/config/bootstrap/v3"
+)
+
+func TestDumper(t *testing.T) {
```

Comment:
I don't see these tests being run in a github action; is this intended?

Reply:
This is the test being run:
`ok  	github.com/twosigma/envoy-viz/configreader	1.037s`
If one fails it will print the log from the test but when it succeeds this is all it prints

Reply:
I see it in the output now, didn't realize that `make clean && make all` also runs the test, but the makefile change makes that clear, 

Have you confirmed that the github action run fails if the tests fail, and that the failure output gives you all the information you would like when debugging the test?

Reply:
Nice, then everything looks good to me